### PR TITLE
Build docker image via travis#260

### PR DESCRIPTION
**Pull request by kleinchang**
#### Description

Tell reviewer something about this change 

#### QA URL

https://demo-roster.efcsydney.org/...

#### Acceptance Criteria

- [ ] Ensure that ...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,21 +35,23 @@ script:
...
 - openssl aes-256-cbc -K $encrypted_42e0c1fefb4c_key -iv $encrypted_42e0c1fefb4c_iv
   -in deploy_rsa.enc -out ~/.ssh/key/efcsydney.pem -d
 - eval "$(ssh-agent -s)"
 - chmod 400 ~/.ssh/key/efcsydney.pem
+- docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
 deploy:
 - provider: script
   skip_cleanup: true
-  script: sh -x deploy.sh qa
+  script: sh -x deploy.sh qa | docker-compose -f docker/docker-compose.yml build efc-qa && docker push kleinchang/efcsydney-roster:qa
```
Comment (liam-lai) [inline .travis.yml]: why push `|`

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+sudo: required
```
Comment (liam-lai) [inline .travis.yml]: any reason for this?

File: .travis.yml (hunk 1 above)
Comment (kleinchang) [reply .travis.yml]: that was a decision from last discussion with Brandon, my understanding is Brandon planned to pull this image from my repository into ECS. If there is an easier way to deploy image to ECS then pushing to docker hub might not be necessary. Any suggestion?

File: .travis.yml (hunk 2 above)
Comment (kleinchang) [reply .travis.yml]: This is from some sample code I found, let me confirm if that's really necessary.

File: .travis.yml (hunk 1 above)
Comment (BrandonTsai) [reply .travis.yml]: you can use ";" to connect multiple commands in one line.  such as:
sh -x deploy.sh qa ; docker-compose -f docker/docker-compose.yml build efc-qa && docker push kleinchang/efcsydney-roster:qa

Comment (kleinchang) [reply .travis.yml]: Okay thanks for the clarification, updated.

**Pull request merged**